package.json

```diff
@@ -29,6 +29,7 @@
     "heroku-client": "3.0.2",
     "mocha": "3.2.0",
     "nock": "9.0.11",
+    "np": "^2.15.0",
```

Comment:
We should save the exact version here, no? Also @dickeyxxx introduced yarn.lock in e8dc71e9749e138710cec3afe016397dfa3cfeb3 -- we should probably use yarn to add this dep and updated the lockfile.